android/build.gradle

```diff
@@ -16,19 +16,22 @@ if (flutterRoot == null) {
 
 buildscript {
     ext.kotlin_version = '1.8.0'
-    ext.doordeck_android_sdk_version = '2.0.16'
+    ext.doordeck_android_sdk_version = '2.2.0'
     repositories {
+        mavenLocal()
         google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.4.2'
+        classpath 'com.android.tools.build:gradle:8.1.1'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }
 
 rootProject.allprojects {
     repositories {
+        mavenLocal()
         google()
         jcenter()
         maven { url 'https://www.jitpack.io' }
```

Comment:
We've really gotta get the doordeck sdk in maven central... we have creds already setup

Reply:
My bad, test purposes to test locally beforehand :)